src/components/shared/checkbox.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+export class Checkbox extends React.Component {
```

Comment:
Why is this a class component? Why not make it a functional one? What's the need behind making it a class?

Reply:
Fixed it is a functional component.